cypress/integration/ui/create_dashboard/index.js

```diff
@@ -0,0 +1,111 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+const UID_LENGTH = 11
+
+const dashboardTitle = new Date().toString()
+
+const ROUTE_EDIT = 'edit'
+const ROUTE_NEW = 'new'
+const ROUTE_PRINTLAYOUT = 'printlayout'
+const ROUTE_PRINTOIPP = 'printoipp'
+const nonViewRoutes = [
+    ROUTE_NEW,
+    ROUTE_EDIT,
+    ROUTE_PRINTLAYOUT,
+    ROUTE_PRINTOIPP,
+]
+
+const getRouteFromHash = hash => {
+    const lastSlashIdx = hash.lastIndexOf('/')
+    return hash.slice(lastSlashIdx + 1)
+}
+
+const toggleShowMoreButton = () => {
```

Comment:
Extract this to a separate file containing all the helper functions for the toolbar buttons?

Reply:
I was waiting to do this kind of things until I had more tests and see what kind of helper files make sense